fake-toolchain/Unix/cc

```diff
@@ -9,6 +9,6 @@ elif [ "$1" = "-v" ] || [ "$1" = "--version" ]; then
     $YCM_CONFIG_GEN_CC_PASSTHROUGH $@
 
 else
-    echo "$@" >> $YCM_CONFIG_GEN_CC_LOG
+	echo "YCM_CONFIG_GEN_PWD=$(pwd) $@" >> $YCM_CONFIG_GEN_CC_LOG
```

Comment:
The flags are split up / tokenized by `split_flags()`, which uses `unbalanced_quotes()` to concatenate adjacent words containing quotes. Since `YCM_CONFIG_GEN_PWD=` is checked for in a flag, it will have the same logic applied to it.